pkg/config/generate.go

```diff
@@ -18,6 +19,49 @@ import (
 	"github.com/jim-minter/azure-helm/pkg/tls"
 )
 
+// ORIGIN Images
```

Comment:
this comment makes me think that a simplifying refactor is needed on the code, not an additional comment.

Reply:
I was thinking to keep this as a reference in the comments as list here is far from what we use. I would keep the comments and refactor code :)